src/inapp/request.ts

```diff
@@ -37,100 +30,9 @@ type RequestMessagesProps = {
   payload: InAppMessagesRequestParams;
 };
 
-export const requestMessages = async ({ payload }: RequestMessagesProps) => {
-  /** @note TBD: Caching implementation and associated parameter will be enabled once new endpoint is ready */
+export const requestMessages = async ({ payload }: RequestMessagesProps) =>
+  /** @note TBD: Caching implementation and associated parameter
+   * will be enabled once new endpoint is ready */
   // if (!options?.useLocalCache) return await requestInAppMessages({});
   /** @note Always early return until then */
-  return await requestInAppMessages({ payload });
```

Comment:
Everything after the early return here is valid and tested code, ready to be used when we have a new endpoint to handle cached messages. Think the linter doesn't like having unreachable code, so maybe we can just comment this out with a note? Or refactor to pull this out to a util?